src/app.py

```diff
@@ -76,7 +74,7 @@ def assaytypes():
     primary = None
     simple = False
     for key, val in request.args.items():
-        print(f'{key}:{val}')
+        #print(f'{key}:{val}')
```

Comment:
Delete, or change to an `info` or `debug` log?

Reply:
Yeah, delete.